srothenburger1/phlex
# Fix Navbar toggling logic

#### What does this PR do?
- modifies the landing page search event handler name.
- modifies the logic for toggling Navbar style for the App.

Description of Task to be completed?

#### How should this be manually tested?
- Pull the branch
- run `npm install` to install dependencies
- run `npm start` to start the application
- Navbar on the Landing page, should be differ from that of other pages.

####  Any background context you want to provide?

N/A

 #### What are the relevant Phlex Board stories?

[#card-18997329](https://github.com/srothenburger1/phlex/projects/1#card-18997329)

####  Any Screenshots?
N/A


 #### Any Questions?
N/A
